sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -102,6 +103,8 @@ class SparkSession private(
         new SparkSessionExtensions), Map.empty)
   }
 
+  val sessionUUID: String = UUID.randomUUID.toString
```

Comment:
shall we mark it as `private[sql]` if we don't want to expose it as a public API?